gui/wxpython/location_wizard/wizard.py

```diff
@@ -482,7 +487,10 @@ def __init__(self, wizard, parent):
                                   style=wx.TE_PROCESS_ENTER)
 
         # projection list
-        self.projlist = ItemList(self, data=self.parent.projdesc.items(),
+        data = list()
+        for item in self.parent.projdesc.items():
+            data.append(item)
+        self.projlist = ItemList(self, data=data,
```

Comment:
why not shorter `list(self.parent.projdesc.items())` ?

Reply:
Good point, fixed.